usr/share/rear/restore/DP/default/450_restore_via_gui.sh

```diff
@@ -7,13 +7,13 @@ Log "Request for a manual restore via the GUI"
 
 LogUserOutput "
 **********************************************************************
-**  Please try to restore the backup from Data Protector GUI!
-**  Make sure you select \"overwrite\" (destination tab) and make the
-**  new destination $TARGET_FS_ROOT.
-**  When the restore is complete press ANY key to continue!
+* The Data Protector client is available on the network. Restore a
+* backup from the Data Protector GUI. Make sure you select \"overwrite\"
```

Comment:
Only a side-note hint for possibly better readable code:
Instead of escaped double quotes within double quotes like
```
echo "Run \"program\" to do..."
```
I personaly prefer unescaped single quotes within double quotes
```
echo "Run 'program' to do..."
```
because this makes the code look less complicated.

Reply:
I went to use `>string<` to solve this problem. After all the point is only to show where the external program/string/info starts and ends